noir-projects/noir-protocol-circuits/crates/types/src/proof/verification_key.nr

```diff
@@ -12,32 +12,6 @@ pub struct VerificationKey<let N: u32> {
     pub hash: Field,
 }
 
-impl<let M: u32> Serialize for VerificationKey<M> {
```

Comment:
These implementations were unused (everything compiled without them) so I dropped them. It's straightforward to slap `#[derive(Serialize)]` on the struct later on if needed.